runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -450,15 +456,11 @@ private void rebuild()
 		}
 		else
 		{
-			int start = 0;
-			if (sessionRecords.size() > MAX_LOOT_BOXES)
-			{
-				start = sessionRecords.size() - MAX_LOOT_BOXES;
-			}
-			for (int i = start; i < sessionRecords.size(); i++)
-			{
-				buildBox(sessionRecords.get(i));
-			}
+			sessionRecords.stream()
+				.sorted(Collections.reverseOrder())
+				.filter(r -> !hideIgnoredItems || !plugin.isEventIgnored(r.getTitle()))
```

Comment:
I would maybe add a comment indicating why this is checking ignored (since buildBox also checks which looks unnecessary) as it is subtle.